### PR TITLE
docs: Add quota removal step to namespace apply

### PR DESCRIPTION
**Pull request by krishicks**
This updates `quota init` to provide an unlimited quota by default, which is then referenced in the docs for how to remove a quota. The [Learn docs](https://learn.hashicorp.com/tutorials/nomad/quotas) would be out of date with this change, but maybe should also be updated to include similar quota removal steps.

### CODE REVIEW

File: command/quota_init.go [hunk 1]
```diff
@@ -106,30 +106,22 @@ func (c *QuotaInitCommand) Run(args []string) int {
...
 
-# Create a limit for the global region. Additional limits may
-# be specified in-order to limit other regions.
 limit {
     region = "global"
-    region_limit {
-        cpu = 2500
-        memory = 1000
-    }
+    region_limit {}
```
Comment (tgross) [inline command/quota_init.go:113]: I agree this is the right value to have, but for purposes of using `nomad quota init` to get started, maybe we can comment-out the other value as an example? Similar to what we do with `nomad job init`

File: website/pages/docs/commands/namespace/apply.mdx [hunk 2]
```diff
@@ -42,3 +42,21 @@ Create a namespace with a quota
 $ nomad namespace apply -description "Prod API servers" -quota prod api-prod
 Successfully applied namespace "api-prod"!
 ```
+
+Remove a quota from a namespace
```
Comment (tgross) [inline website/pages/docs/commands/namespace/apply.mdx:46]:
```suggestion
## Remove a quota from a namespace
```

File: website/pages/docs/commands/namespace/apply.mdx [hunk 3]
```diff
@@ -42,3 +42,21 @@ Create a namespace with a quota
...
+
+Quotas cannot be removed from a namespace once they have been applied, only replaced. To effectively remove a quota, create and apply an unlimited quota.
```
Comment (tgross) [inline website/pages/docs/commands/namespace/apply.mdx:48]: We don't have a linter for it for some reason (not sure there is one?), but generally we hard-wrap markdown lines.

File: website/pages/docs/commands/namespace/apply.mdx [hunk 4]
```diff
@@ -42,3 +42,21 @@ Create a namespace with a quota
...
+$ cat spec.hcl
+name = "default-quota"
+description = "Unlimited default"
+
+limit {
+    region = "global"
+    region_limit {}
+}
+$ nomad quota apply spec.hcl
+$ nomad namespace apply -quota default-quota api-prod
```
Comment (tgross) [inline website/pages/docs/commands/namespace/apply.mdx:62]: Let's close this code block.

**Review by tgross — CHANGES_REQUESTED** [review]

File: website/pages/docs/commands/namespace/apply.mdx (hunk 3 above)
Comment (cgbaker) [inline website/pages/docs/commands/namespace/apply.mdx:48]: is that true? 

```
$ nomad namespace apply -quota= test
Successfully applied namespace "test"!
~
$ nomad namespace inspect test
{
    "CreateIndex": 11,
    "Description": "",
    "ModifyIndex": 19,
    "Name": "test",
    "Quota": ""
}
```

**Review by cgbaker — CHANGES_REQUESTED** [review]
not sure about the premise

File: website/pages/docs/commands/namespace/apply.mdx (hunk 3 above)
Comment (tgross) [reply website/pages/docs/commands/namespace/apply.mdx:48]: I know we don't like linking off to ENT but: https://github.com/hashicorp/nomad-enterprise/issues/414

Comment (krishicks) [reply website/pages/docs/commands/namespace/apply.mdx:48]: @tgross Thoughts? Would it be better to document the `quota= ` behavior for removal?

Comment (tgross) [reply website/pages/docs/commands/namespace/apply.mdx:48]: Re-read my own issue... Ah, yeah that seems better. It's that `nomad quota delete` doesn't work quite as expected. So yeah let's document @cgbaker's solution.

File: website/pages/docs/commands/namespace/apply.mdx (hunk 2 above)
Comment (krishicks) [reply website/pages/docs/commands/namespace/apply.mdx]: I looked at a bunch of other docs pages and found it's most common not to h2 these, but rather list each example under a single h2. Examples header.

File: website/pages/docs/commands/namespace/apply.mdx (hunk 3 above)
Comment (krishicks) [reply website/pages/docs/commands/namespace/apply.mdx]: Done!

File: website/pages/docs/commands/namespace/apply.mdx (hunk 2 above)
Comment (tgross) [reply website/pages/docs/commands/namespace/apply.mdx]: Ah, this _is_ an example, not a separate section. Makes sense.

**Review by cgbaker — APPROVED** [review]
👍

**Review by tgross — APPROVED** [review]

**Pull request merged**